runner/scheduler.py

```diff
@@ -313,43 +285,49 @@ def initialise_result(self, example: TestExample) -> tuple[TestResult, bool]:
             result.load(self.agent.name)
         return result, skip
 
-    def run_interleaved_group(self, test_group: list[TestExample]):
-        if not self.config.continuous_conversation:
-            self.agent.reset()
-
+    def run_tests(self):
         self.result_callbacks = []
-        self.group_master_log = []
+        self.master_log = []
+        datasets_run = set()
         results = dict()
-        colour_print("Green", f"Group has {len(test_group)} tests.")
         finished = 0
-        for example in self.iter_tests(test_group):
+        for example in self.iter_tests(self.tests):
+            self.progress_dialog.notify_running(example)
+
             skip = False
             if example.unique_id not in results:
                 result, skip = self.initialise_result(example)
                 results[example.unique_id] = result
                 example.finished = skip
 
+                # Ask the agent to ignore any previous related information.
```

Comment:
Is this a reset of the test after a resume? Even before we have any guarantees that the agent saves information?